duet/explores/desktop_activation.explore.lkml

```diff
@@ -21,8 +21,8 @@ explore: desktop_activation {
         DATE(DATE_ADD(
           DATE({% date_end desktop_activation.date %}), INTERVAL DATE_DIFF(DATE({% date_start desktop_activation.date %}), DATE({% date_end desktop_activation.date %}), DAY) DAY)),
           DATE({% date_end desktop_activation.date %})),
-      -- if the most recent week is to be ignored, shift date range by 8 days
-      INTERVAL IF({% parameter desktop_activation.ignore_most_recent_week %}, 8, 0) DAY)
+      -- if the most recent week is to be ignored, shift date range by 9 days
```

Comment:
What's the motivation behind changing from 8 days to 9 days?

Reply:
The 7 day activation can lag 9 days behind, then the value in the graph is set to 0 which might cause some confusion. Shifting by 9 days also seems to be what is done in the previous Numbers That Matter dashboard